libs/ha/ha_common_libs_k8s.py

```diff
@@ -746,7 +745,7 @@ def start_random_mpu(self, event, s3_data, bucket_name, object_name, file_size,
             LOGGER.info("Response: %s", res)
             mpu_id = res[1]["UploadId"]
             LOGGER.info("Multipart Upload initiated with mpu_id %s", mpu_id)
-        except (Exception, CTException) as error:
+        except CTException as error:
             LOGGER.error("Failed mpu due to error %s. Exiting from background process.", error)
```

Comment:
Here also, please revert the changes.
Or instead of Exception add BaseException but removing it may cause failures.